src/lib.rs

```diff
@@ -86,6 +86,8 @@ pub use crate::{
     transaction::{DropBehavior, Savepoint, Transaction, TransactionBehavior},
     types::ToSql,
 };
+#[cfg(feature = "polars")]
+pub use polars_dataframe::{Polars, PolarsDataFrame};
 
 // re-export dependencies from arrow-rs to minimise version maintenance for crate users
 pub use arrow;
```

Comment:
Do you think we also need to re-export polars like we do for arrow?

https://github.com/wangfenjin/duckdb-rs/pull/93

Reply:
Yes. I will add it.